cores/nRF5/wiring.c

```diff
@@ -34,6 +34,7 @@ void init( void )
   NRF_CLOCK->LFCLKSRC = (uint32_t)((CLOCK_LFCLKSRC_SRC_RC << CLOCK_LFCLKSRC_SRC_Pos) & CLOCK_LFCLKSRC_SRC_Msk);
   NRF_CLOCK->TASKS_LFCLKSTART = 1UL;
 
+  NRF_RTC1->INTENSET = 2;
```

Comment:
What do you think about using the `RTC_INTENSET_OVRFLW_Msk` constant here instead of 2?

Declared:

 * https://github.com/sandeepmistry/arduino-nRF5/blob/master/cores/nRF5/SDK/components/device/nrf51_bitfields.h#L5465
 * https://github.com/sandeepmistry/arduino-nRF5/blob/master/cores/nRF5/SDK/components/device/nrf52_bitfields.h#L11483